org-code-javabuilder/protocol/src/main/java/org/code/protocol/LoggerNames.java

```diff
@@ -0,0 +1,5 @@
+package org.code.protocol;
+
+public class LoggerNames {
+  public static final String MAIN_LOGGER = "mainLogger";
```

Comment:
I don't know how sophisticated we want to make the logging, but IIRC, in other systems I've worked on, we used the fully qualified class name as the logger name to create a hierarchy of loggers.  This allowed for very granular control of the logging verbosity for different parts of the system.

Reply:
In the future, we may want something like that, but for now, I think we can keep it simple